scripts/build-icons.js

```diff
@@ -177,41 +185,45 @@ for (const icon of whitelist) {
 }\n`)
 
     // Generate symbol image
-    continue // It's not ready yet
     run(`mkdir -p ${sfolder}`)
     fs.writeFileSync(`${sfolder}/${name}.svg`,
-`<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="-192 -192 2304 2304">
+`
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
  <g id="Guides">
-  <line id="Baseline-S" x1="0" x2="1920" y1="1540" y2="1540"/>
-  <line id="Capline-S" x1="0" x2="1920" y1="380" y2="380"/>
-  <line id="Baseline-M" x1="0" x2="1920" y1="1540" y2="1540"/>
-  <line id="Capline-M" x1="0" x2="1920" y1="380" y2="380"/>
-  <line id="Baseline-L" x1="0" x2="1920" y1="1540" y2="1540"/>
-  <line id="Capline-L" x1="0" x2="1920" y1="380" y2="380"/>
-  <rect id="left-margin" x="-192" y="-192" width="192" height="2304" />
-  <rect id="right-margin" x="1920" y="-192" width="192" height="2304" />
+  <line id="Baseline-S" x1="0" x2="24" y1="18" y2="18"/>
+  <line id="Capline-S"  x1="0" x2="24" y1="6" y2="6"/>
+  <line id="Baseline-M" x1="0" x2="24" y1="18" y2="18"/>
+  <line id="Capline-M"  x1="0" x2="24" y1="6" y2="6"/>
+  <line id="Baseline-L" x1="0" x2="24" y1="18" y2="18"/>
+  <line id="Capline-L"  x1="0" x2="24" y1="6" y2="6"/>
+  <line id="left-margin" x1="0" x2="0" y1="0" y2="24"/>
+  <line id="right-margin"  x1="24" x2="24" y1="0" y2="24"/>
```

Comment:
looks like there's an extra space after the id here. if you are trying to line things up you actually want the extra space after left-margin.